articles/vmware/vmw-ref-vcd-91.md

```diff
@@ -50,6 +50,8 @@ You should be aware of the following end-of-life and end-of-support warnings for
 
 For a list of known issues with vCloud Director 9.1, see the [*vCloud Director 9.1 for Service Providers Release Notes*](https://docs.vmware.com/en/vCloud-Director/9.1/rn/rel_notes_vcloud_director_91.html#knownissues).
 
+Only vShield Edge Gateways which have been converted to Advanced Gateway UI can be managed in the Enhanced HTML-5 tenant portal. To change settings on non-Advanced Gateway Edges, it is necessary to select "See this page in vCloud Director Web Console" in the vApp or Virtual Machine section of the HTML-5 portal to view the previous vCloud web portal.
```

Comment:
Change this to:

You can only manage advanced gateways in the new vCloud Director 9.1 tenant portal. If you have not yet converted your edge to an advanced gateway, you must first switch to the vCloud Director web console to manage the edge.  For more information about converting your edge to an advanced gateway, see [*How to convert your edge to an advanced gateway*](vmw-how-convert-edge.md). For more information about switching to the web console, see [*How to switch to the vCloud Director web console from the tenant portal*](vmw-how-switch-web-console.md). 

I'd also suggest putting this paragraph first and then change the beginning of the other paragraph to:

For a list of other known issues...